upstream-community-operators/aws-service/aws-service.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,194 @@
+  provider:
+    name: AWS Labs
+  labels:
+    name: aws-service-operator
+  selector:
+    matchLabels:
+      name: aws-service-operator
+  links:
+  - name: AWS Operator Source Code
+    url: https://github.com/awslabs/aws-service-operator
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: false
+  - type: MultiNamespace
+    supported: false
+  - type: AllNamespaces
+    supported: true
+  install:
+    strategy: deployment
+    spec:
+      clusterPermissions:
+      - serviceAccountName: aws-service-operator
+        rules:
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          - pods
+          - configmaps
+          - services
+          - events
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+          - update
+          - patch
+        - apiGroups:
+          - apiextensions.k8s.io
+          resources:
+          - customresourcedefinitions
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - delete
+        - apiGroups:
+          - service-operator.aws
+          resources:
+          - "*"
+          verbs:
+          - "*"
+      deployments:
+      - name: aws-service-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: aws-service-operator
+          template:
+            metadata:
+              annotations:
+                iam.amazonaws.com/role: aws-service-operator
+              labels:
+                app: aws-service-operator
+            spec:
+              serviceAccountName: aws-service-operator
+              containers:
+              - name: aws-service-operator
+                image: awsserviceoperator/aws-service-operator:v0.0.1-alpha4
+                imagePullPolicy: Always
+                env:
+                - name: CLUSTER_NAME
+                  valueFrom:
+                    secretKeyRef:
+                      name: aws-service-operator-config
+                      key: CLUSTER_NAME
+                - name: REGION
+                  valueFrom:
+                    secretKeyRef:
+                      name: aws-service-operator-config
+                      key: REGION
+                - name: ACCOUNT_ID
+                  valueFrom:
+                    secretKeyRef:
+                      name: aws-service-operator-config
+                      key: ACCOUNT_ID
+                args:
+                  - server
+                  - --cluster-name=$(CLUSTER_NAME)
+                  - --region=$(REGION)
+                  - --account-id=$(ACCOUNT_ID)
+  customresourcedefinitions:
+    owned:
+    - name: cloudformationtemplates.service-operator.aws
+      version: v1alpha1
+      kind: CloudFormationTemplate
+      displayName: CloudFormationTemplate
+      description: Represents a resource for managing Cloud Formation Templates.
```

Comment:
s/Cloud Formation Templates/AWS CloudFormation Templates/

Reply:
Resolved in the latest commit.